INSTALL

```diff
@@ -568,6 +574,12 @@ The specific libraries and their options are:
   Options: SUPERLU_OPT, SUPERLU_LIB.
   Versions: SuperLU_DIST >= 5.1.0.
 
+- MUMPS (optional), used when MFEM_USE_MUMPS = YES. Note that MUMPS
```

Comment:
You should mention that it also requires `MFEM_USE_LAPACK=YES`, `MFEM_USE_SCALAPACK=YES`.

Reply:
If LAPACK and ScaLAPACK are needed for MUMPS that does not necessarily mean you need to enable them in MFEM too (by setting `MFEM_USE_LAPACK=YES` and/or `MFEM_USE_SCALAPACK=YES`). What you can do is just use `LAPACK_OPT`, `LAPACK_LIB`, `SCALAPACK_OPT`, and `SCALAPACK_LIB` when defining `MUMPS_OPT` and `MUMPS_LIB`. In other words, the options `MFEM_USE_LAPACK` and `MFEM_USE_MUMPS` should be independent.

Also, I don't think the option `MFEM_USE_SCALAPACK` really does anything, so it should probably be removed. Of course, the variables `SCALAPACK_OPT` and `SCALAPACK_LIB` should stay since they will allow proper configuration of the `MUMPS_*` variables, as mentioned above.

Reply:
There is no MFEM_USE_SCALAPACK option.